sprocs/workspace_hosts_find_terminable.sql

```diff
@@ -15,7 +15,7 @@ BEGIN
             wh.*
         FROM
```

Comment:
we can probably remove the join and group entirely from this query, I think I was originally trying to calculate the number of running workspaces before realizing that there was a `load_count` column